tests/build.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+set -exuo pipefail
+
+# This script basically builds bootc from source using the provided base image,
+# then runs the target tests.
+
+mkdir -p /tmp/tmp-bootc-build
+BOOTC_TEMPDIR="/tmp/tmp-bootc-build"
+
+# Get OS info from TEST_OS env
+OS_ID=$(echo "$TEST_OS" | cut -d '-' -f 1)
+OS_VERSION_ID=$(echo "$TEST_OS" | cut -d '-' -f 2)
+
+# Base image
+case "$OS_ID" in
+    "centos")
+        TIER1_IMAGE_URL="quay.io/centos-bootc/centos-bootc:stream${OS_VERSION_ID}"
```

Comment:
Not new and not directly related to this but we have this legacy of "TIER1" naming - there's only one default image and it's likely to stay that way. I'd probably just say `BASE_IMAGE_URL` going forward for things like this.